profile/README.md

```diff
@@ -7,9 +7,9 @@
     <p> Here, right now on <b> your </b> screen, you are in the presence of the most inclusive <i> Open Source </i> community on the planet! </p>
     <p> This organisation is the home of <i> open source</i>! Make Pull Requests, open Issues, share ideas, help each other and make improvements!</p>
 </div>
-<h2>Suggested steps to get started</h2>
+<h2>Steps to get started</h2>
 <ul>
-    <li>1. Join the EddieHub GitHub Org by raising an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization">issue</a> (you will be sent an invitation)</li>
+    <li>1. Join the EddieHub GitHub Org by raising an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization">issue</a> (you will recieve an invitation)</li>
```

Comment:
```suggestion
    <li>1. Join the EddieHub GitHub Org by raising an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization">issue</a> (you will receive an invitation)</li>
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
sorry for the spelling mistake will correct it now.